packages/eventual-send/src/index.d.ts

```diff
@@ -1,61 +1,69 @@
+// @ts-check
```

Comment:
will this file not be checked without this?

Reply:
> will this file not be checked without this?

Your implication is right: it's unnecessary.

@mhofman and I were experimenting with trying to get the file checked by `tsc` when it really wasn't.  Turns out `"skipLibCheck": false` was the missing requirement.  IOU an issue.

Reply:
Issue: https://github.com/Agoric/agoric-sdk/issues/4735